test-case/manual/check-wov-trigger.md

```diff
@@ -0,0 +1,64 @@
+
+# Wov pipline trigger test
+Wov pipline trigger testing for now.
+
+# Preconditions
+1. Wov-test.sh in wov-script folder.Please check the script before run it to make sure use the right wov hw device id which match your platform.
+2. Test unit support wov test
+
+# Test Description
+* Change to different preamble/history via command worked on wov-test script.
+* Change to fs16/24/32_le format via arecord command 
+
+## Wov Test Case:
+#####1.check WoV with s16_le format 
+1. Run command via "arecord -Dhw:0,99 -M -N -f s16_le -r 16000 -i -vvv -c2 --period-size=68000 ./tmp.wav" 
+2. Clap hands or make some noise to trigger the wov after process starts.
+3. Check the quality of recorded audio files.
+
+#####2.Check WoV with s24_le format 
+1. Run command"arecord -Dhw:0,99 -M -N -f s24_le -r 16000 -i -vvv -c2 --period-size=68000 ./tmp.wav" 
+2. Clap hands or make some noise to trigger the wov after process starts.
+3. Check the quality of recorded audio files.
+
+#####3.Check WoV with s32_le format 
+1. Run command"arecord -Dhw:0,99 -M -N -f s24_le -r 16000 -i -vvv -c2 --period-size=68000 ./tmp.wav" 
+2. Clap hands or make some noise to trigger the wov after process starts.
+3. Check the quality of recorded audio files.
+
+#####4.Check WoV with different buffer sizes(must > 67200) 
+1. Run command"record -Dhw:0,99 -M -N -f s32_le -r 16000 -i -vvv -c2 --buffer-size=68000 ./tmp.wav"
```

Comment:
This size varies with different platform, please add some comments here to mention this.